core/trino-main/src/main/java/io/trino/server/TrinoSystemRequirements.java

```diff
@@ -129,6 +130,20 @@ private static void verifyUsingG1Gc()
         }
     }
 
+    private static void verifyVectorApiEnabled()
+    {
+        try {
+            Class<?> clazz = Class.forName("jdk.incubator.vector.VectorShape");
+            int preferredBits = (int) clazz.getMethod("vectorBitSize").invoke(clazz.getMethod("preferredShape").invoke(null));
```

Comment:
Any reason for this specific choice of class and method to verify the presence of the Vector API?